Auto corrupt cops and higher jail times

ChillerDragon opened the pull request:
When being jailed for spawn blocking pay automatically 500 - 10000 money
to the officer to reduce jail time by 5 minutes.

The amount depends on the current activity of spawnblocking.

Also extend jail time by 5 minutes based on every 100 seconds left escape
time.

fokkonaut commented on src/game/server/entities/character.cpp:
```diff
@@ -544,12 +544,25 @@ void CCharacter::FireWeapon()
-						str_format(aBuf, sizeof(aBuf), "You caught the gangster '%s' (10 minutes arrest)", Server()->ClientName(TargetCID));
+						int Minutes = clamp((int)((pTarget->GetPlayer()->m_EscapeTime / Server()->TickSpeed()) / 100) * 5, 10, 60);
+						int Corrupt = clamp(pTarget->GetPlayer()->m_SpawnBlockScore * 500, 500, 10000);
+						if (m_pPlayer->GetAccID() >= ACC_START && GameServer()->m_Accounts[m_pPlayer->GetAccID()].m_Money > Corrupt)
```
check pTarget's acc, not your own

ChillerDragon replied:
good catch fixed :)

ChillerDragon committed:
Auto corrupt cops and higher jail times

When being jailed for spawn blocking pay automatically 500 - 10000 money
to the officer to reduce jail time by 5 minutes.

The amount depends on the current activity of spawnblocking.

Also extend jail time by 1 minute based on every 100 seconds left escape
time.

ChillerDragon commented:
Force pushed to extend jail time only by 1 minute per 100 seconds escape time and reduce max to 20 minutes.
